test/post-build/integration.test.ts

```diff
@@ -0,0 +1,5 @@
+import '../integration.test'
+
+beforeAll(() => {
+  process.env.POST_BUILD = '1'
```

Comment:
Q: Is `beforeAll` required? not sure what will it effect other tests?
Or do we also need to reset the env afterwards in afterAll?

Reply:
Seems like beforeAll is not necessary, I'll remove it my bad.